pkg/component/worker/kubelet.go

```diff
@@ -82,7 +77,6 @@ func (k *Kubelet) Run() error {
 
 	args := []string{
 		fmt.Sprintf("--root-dir=%s", k.dataDir),
-		fmt.Sprintf("--volume-plugin-dir=%s", k.K0sVars.KubeletVolumePluginDir),
```

Comment:
Unsetting this will now make the default location to change :/ Could we overcome this by setting the default worker profile to have the current default path?